Resources/Prototypes/DeltaV/Entities/Clothing/Neck/ties.yml

```
@@ -52,3 +52,14 @@
     sprite: DeltaV/Clothing/Neck/Ties/greentie.rsi
   - type: Clothing
     sprite: DeltaV/Clothing/Neck/Ties/greentie.rsi
+
+- type: entity  #Delta-V chemist's tie
```

```suggestion
- type: entity
```
No need for a change marker since it is in a DeltaV file.